main.js

```diff
@@ -37,35 +70,6 @@ document.getElementById("pretoria-container").style.visibility = "visible";
 document.getElementById("johannesburg-container").style.visibility = "hidden";
```

Comment:
Can this not be an exported function from the DOM manipulation handler so you don't have to know implementation details in your orchestrator?

Reply:
Created a function `displaySelectedCity` in domManipulation.js which handles this now. 
main.js calls this fucntion with "pretoria" to ensure Pretoria is the default selected city.